packages/zowe-explorer/package.json

```diff
@@ -2041,7 +2041,7 @@
     ]
   },
   "scripts": {
-    "build": "pnpm clean:bundle && pnpm license && webpack --mode development",
+    "build": "pnpm clean:bundle && (pnpm madge || true) && pnpm license && webpack --mode development",
```

Comment:
I assume this is temporary 😋

Reply:
Yes, planning to fix this on the [fix/circular-imports](https://github.com/zowe/vscode-extension-for-zowe/tree/fix/circular-imports) branch 🙂